pkg/collector/corechecks/net/ntp_test.go

```diff
@@ -343,3 +343,30 @@ func TestDefaultHostConfig(t *testing.T) {
 
 	assert.Equal(t, expectedHosts, ntpCheck.cfg.instance.Hosts)
 }
+
```

Comment:
Could you add a test with the port set as a string to "define" the desired behavior in this case ?